cli/graph_util.rs

```diff
@@ -42,11 +35,8 @@ pub fn contains_specifier(
 pub enum ModuleEntry {
   Module {
     code: Arc<str>,
-    maybe_parsed_source: Option<ParsedSource>,
```

Comment:
I think I placed this here a few releases back which caused the memory usage regression. I opened https://github.com/denoland/deno/issues/15531